db/external_sst_file_ingestion_job.cc

```diff
@@ -469,8 +469,10 @@ Status ExternalSstFileIngestionJob::Run() {
         f.fd.GetNumber(), f.fd.GetPathId(), f.fd.GetFileSize(),
         f.smallest_internal_key, f.largest_internal_key, f.assigned_seqno,
         f.assigned_seqno, false, f.file_temperature, kInvalidBlobFileNumber,
-        oldest_ancester_time, current_time, f.file_checksum,
-        f.file_checksum_func_name, f.unique_id);
+        oldest_ancester_time, current_time,
+        f.picked_level == 0 ? versions_->NewL0EpochNumber()
```

Comment:
[TODO] Should assign epoch_number based on seqno within a bulk of ingested files.